src/App.js

```diff
@@ -2,24 +2,25 @@ import { useState, useEffect } from "react"
 import TodoList from "./TodoList"
 import AddTodoForm from "./AddTodoForm"
 
+const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`
+const myInit = {
+  method: "GET",
+  headers: {
+    Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+  },
+}
```

Comment:
I like how you moved these into their own variable 💯

Reply:
Thanks! it makes much easier to read!